packages/cli/__tests__/zoslogs/__integration__/list/__snapshots__/cli.zos-logs.list.logs.integration.test.ts.snap

```diff
@@ -0,0 +1,139 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`zos-logs list should display the help 1`] = `
+"================Z/OS LOG LIST LOGS HELP===============
+
+ COMMAND NAME
+ ------------
+
+   logs | logs
+
+ DESCRIPTION
+ -----------
+
+   List z/OS operlog or syslog within a time range.
+
+   Use this operation to get the z/OS logs. It invokes the MVS operations log
+   (OPERLOG) function of the MVS console services component to get the sysplex-wide
+   merged message logs
+
+   Execute 'zowe zos-logs list logs' will by default return logs from current time
+   and backwarding to 10 minutes before
```

Comment:
I'd suggest some slight grammatical changes here: `Execute` -> `executing` and `backwarding` to `backwards`.